docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -7899,6 +7899,46 @@ end
 
 * https://rubystyle.guide#no-self-unless-required
 
+== Style/RedundantSelfAssignment
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| No
+| Yes (Unsafe)
+| 0.90
+| -
+|===
+
+This cop checks for places where redundant assignments are made for in place
+modification methods.
+
+This cop is marked as unsafe, because it can produce false positives for
+user defined methods having one of the expected names, but not modifying
+its receiver in place.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+args = args.concat(ary)
+hash = hash.merge!(other)
+
+# good
+args.concat(foo)
+args += foo
```

Comment:
I think you forgot the bad example for this.

Reply:
This and the previous line are both good examples for `args = args.concat(ary)` (couple of lines above)

Reply:
Ah, my bad!